Examples/SwiftUI Example/Beacon SwiftUI Example.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,362 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 52;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		3F9A5B2D269E0BC8000AFD6A /* SettingsView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3F9A5B2C269E0BC8000AFD6A /* SettingsView.swift */; };
+		3FD50DDF269E053A00174289 /* Beacon_SwiftUI_ExampleApp.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FD50DDE269E053A00174289 /* Beacon_SwiftUI_ExampleApp.swift */; };
+		3FD50DE1269E053A00174289 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FD50DE0269E053A00174289 /* ContentView.swift */; };
+		3FD50DE3269E053C00174289 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 3FD50DE2269E053C00174289 /* Assets.xcassets */; };
+		3FD50DE6269E053C00174289 /* Preview Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 3FD50DE5269E053C00174289 /* Preview Assets.xcassets */; };
+		3FE2EACB26A8715C00A122F6 /* Beacon-iOS in Frameworks */ = {isa = PBXBuildFile; productRef = 3FE2EACA26A8715C00A122F6 /* Beacon-iOS */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXFileReference section */
+		3F9A5B2C269E0BC8000AFD6A /* SettingsView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = SettingsView.swift; sourceTree = "<group>"; };
+		3FD50DDB269E053A00174289 /* Beacon SwiftUI Example.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "Beacon SwiftUI Example.app"; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
🔍 This is a nitpick... so I hate to even mention it... but it would nice if folder structure wise (doesn't have to extend beyond that) the word "example" was either first or last for both. Right now we have "Example Obj-C" and "Swift UI Example". Would prefer "Example Obj-C" and "Example Swift UI" or vice versa with "Example" at the end. 
But definitely a nitpick so if it's too much of a bother feel free to ignore.